cluster/examples/kubernetes/ceph/object-test.yaml

```diff
@@ -22,3 +22,8 @@ spec:
     port: 80
     securePort:
     instances: 1
+  healthCheck:
+    # Change pod liveness probe, it work grw daemon
```

Comment:
Instead of object-test.yaml, the example would be better in object.yaml.

Reply:
nit : s/grw/rgw

Reply:
Done.